Dockerfile

```diff
@@ -6,4 +6,4 @@ COPY . ./
 
 RUN swift package resolve
 RUN swift package clean
-CMD swift test --parallel
+CMD swift test --parallel -v
```

Comment:
how do you feel about verbose test logging?

Reply:
Sounds good to me. Didn't realize it was available.